app/services/repository_service.py

```
@@ -0,0 +1,46 @@
+        total_count = await self.repo_store.count_by_user(user.sub)
+
+        if total_count == 0:
+            return total_count, []
+
+        repos = await self.repo_store.get_all_by_user(
+            user.sub, pagination.offset, pagination.limit
+        )
+
+        token_ids = {str(r.token_id) for r in repos if r.token_id}
+        labels = await self.gl_store.get_git_hosting_map_by_token_id(token_ids)
+        label_map = {str(l["id"]): l["git_hosting"] for l in labels}
```

Ruff (0.11.9)

Ambiguous variable name: `l`

(E741)